cmd/minishift/cmd/delete.go

```diff
@@ -41,16 +44,23 @@ var (
 
 	forceMachineDeletion bool
 	clearCache           bool
+	deleteAll            bool
 )
 
 func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
-	util.ExitIfUndefined(api, constants.MachineName)
+	if !deleteAll {
```

Comment:
Let's revert this and go for the 'minishift profile delete' as discussed. As a first cut we can ignore requests for 'minishift profile delete minishift' and display an appropriate message.

Reply:
Done.